webapp/channels/src/components/external_image/external_image.test.tsx

```diff
@@ -23,7 +23,7 @@ describe('ExternalImage', () => {
     };
 
     test('should render an image', () => {
-        const wrapper = shallow<ExternalImage>(<ExternalImage {...baseProps}/>);
+        const wrapper = shallow<typeof ExternalImage>(<ExternalImage {...baseProps}/>);
```

Comment:
I think you won't need the typing (and I think it is not 100% correct, but not sure).
```suggestion
        const wrapper = shallow(<ExternalImage {...baseProps}/>);
```

Reply:
I am getting this error when I remove typeof

```
ExternalImage' refers to a value, but is being used as a type here. Did you mean 'typeof ExternalImage'?ts(2749)

```

Reply:
Yeah, my proposal is to remove everything inside the `<>`, not only the typeof.

Reply:
My b. Brain was not working. Changes made